server/src/QRServer/db/migrations.py

```diff
@@ -54,6 +60,39 @@ async def execute_migrations(c):
             " add column discord_user_id varchar"
         )
         await _set_version(c, 5)
+    if version <= 5:
+        await c.execute(
+            "create table rankings ("
+            "  year integer,"
+            "  month integer,"
+            "  position integer,"
+            "  user_id varchar,"
+            "  wins integer,"
+            "  total_games integer,"
+            "  primary key (year, month, position),"
+            "  foreign key(user_id) references users (id)"
+            ")"
+        )
+        await c.execute(
+            "select started_at"
+            " from matches"
+            " order by started_at asc"
+            " limit 1"
+        )
+        row = await c.fetchone()
+        if row:
+            first_timestamp = row[0]
+            first_dt = dt.fromtimestamp(first_timestamp)
+            dates_range = utils.make_month_dates_range(start_date=first_dt, end_date=dt.now())
+
+            for date in dates_range:
+                start_date, end_date = utils.make_month_dates(date.month, date.year)
+
+                ranking_entries = await dbconnector._generate_ranking(start_date, end_date)
```

Comment:
Let's copy the query here. That will prevent errors in the future if we decide to change the behavior of this method. That will also not require passing dbconnector

Reply:
Done. Repeated the above tests (all of them), still works as expected

Reply:
Where did you copy the query? I can't see it

Reply:
![image](https://github.com/user-attachments/assets/be163a21-abb4-4acd-b1bd-15fe08da41aa)